docs/dataframe_api.ipynb

```diff
@@ -0,0 +1,676 @@
+{
```

Comment:
<p>I don't think the comment in the code is necessary here (it is duplicating the text above it)</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/tensorflow/tensorboard/pull/3744/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='9'/>

Reply:
Done.